src/components/PWAPrompt.js

```diff
@@ -38,9 +38,9 @@ const PWAPrompt = ({
     }
   }, [isVisible]);
 
-  const isiOS13 = /OS 13/.test(window.navigator.userAgent);
+  const isiOS13AndUp = /OS (13|14)/.test(window.navigator.userAgent);
```

Comment:
Can probably change this to 1[3-9] to future proof, but this is fine more immediately :)